apps/portal/.env.example

```diff
@@ -0,0 +1 @@
+SERVER_PORT=5000
```

Comment:
when we dockerize this do we have control over API URL? Which API url should this portal use?

Reply:
No, API end point should already be in repository inside `environment.prod.ts` file
(which ever is configured as production in angular.json)

Reply:
https://github.com/OsmosysSoftware/osmo-notify/blob/main/apps/portal/src/environments/environment.prod.ts#L5

![image](https://github.com/OsmosysSoftware/osmo-notify/assets/16570290/bef5d44e-f9b2-4063-bb3b-e47e619026e9)


I see this right now. So it won;t work right if we deploy as docker on docker tower.

Reply:
No, it should have been production API in environment.prod.ts

Reply:
![image](https://github.com/OsmosysSoftware/osmo-notify/assets/14059393/42037a37-c14f-40ab-b578-20912853df5b)
![image](https://github.com/OsmosysSoftware/osmo-notify/assets/14059393/6ce0e61d-cb4d-4ed8-9091-a5260e9bd5ae)

Reply:
My question is why not add a commit since we already know the production url like we did in JLR.

Reply:
Okay will add here itself.

Reply:
Updated production url.
To Check
1. Production Backend is updated with latest GraphQL API.
2. Check for CORS Error if present.